### PR TITLE
Update Laptop

### PR DESCRIPTION
**Pull request by pascalandermatt**

### CODE REVIEW

**Comment by pascalandermatt** [comment]
I don't know, how to run the cli

**Comment by felixkubli** [comment]
Code looks ok, works and can be merged

File: lib/renuo/cli/app/update_laptop.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  def countdown(n)
+    n.downto(1) do |i|
+      say "Reboot in #{i} Seconds (Press Ctrl + C to Stop)".red
+      say `sleep 1`
+    end
+  end
+end
+
+# Colorized Output
+class String
```
Comment (lukaselmer) [inline lib/renuo/cli/app/update_laptop.rb]: https://s-media-cache-ak0.pinimg.com/564x/e8/fa/96/e8fa962b570ecc85671814ea1957d7c2.jpg

Comment (lukaselmer) [inline lib/renuo/cli/app/update_laptop.rb]: Let's discuss this :)

Comment (felixkubli) [inline lib/renuo/cli/app/update_laptop.rb]: ok 👍

**Comment by lukaselmer** [comment]
@pascalandermatt we don't need this branch anymore, correct? can you remove it? thanks!

**Pull request closed without merging**